OneCommunityGlobal/HighestGoodNetworkApp
# Changhao-added new unit tests for weeklysummariesreport.test.jsx

# Description
Update: error are now fixed
added new unit tests for weeklysummariesreport.test.jsx, please ignore prexisting err caused by `EditableInfoModal` and scrow down to see the test results.

## How to test:
1. check into current branch
2. do `npm install` and `npm test WeeklySummariesReport.test.js` to run this PR locally
3. verify all 10 unit tests are passed


## Screenshot:
<img width="833" alt="Screenshot 2024-02-06 at 1 14 21 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/42322bfb-5c46-4432-a2e7-a0d6939e14c4">
